tests/test_exceptions.py

```diff
@@ -490,3 +492,17 @@ def test_BinaryPrefixReplacementError(self):
           original data Length: 1404
           new data length: 1104
         """).strip()
+
+    @patch("conda.exceptions.os.isatty", return_value=True)
+    def test_PackagesNotFoundError_use_only_tar_bz2(self, isatty_mock):
+        note = "use_only_tar_bz2"
+        for use_only_tar_bz2 in (True, False):
+            expected = ["", note][use_only_tar_bz2]
```

Comment:
@jaimergp we could do it this way. We don't have the requirement to "not show the note" when the flag is not set, and this regex matcher is bad at negative testing.

Reply:
As you wish, but I find this way of assigning booleans a bit unreadable. I'd rather see this in code I am maintaining:

```suggestion
            expected = "use_only_tar_bz2" if use_only_tar_bz2 else ""
```

The `raises` + `match` combo is killer though.